src/swap/useSwapQuote.ts

```diff
@@ -70,25 +282,38 @@ const useSwapQuote = () => {
       const swapPrice = useGuaranteedPrice
         ? quote.unvalidatedSwapTransaction.guaranteedPrice
         : quote.unvalidatedSwapTransaction.price
+      const price =
```

Comment:
it is a little unnerving for me to see `swapPrice` and `price`, which are named very similarly, have different units if updatedField is not FROM

Reply:
Yes, we can probably improve that later. Also caused by a feature flag.